Makefile

```diff
@@ -42,15 +45,23 @@ else
 		# uses new 5.2 metamethods and doesn't work on LuaJIT.
 		TESTER = bin/wordgrinder
 	else
-		LUA_INCLUDE := $(INCROOT)/include/lua5.2
-		LUA_LIB := -llua5.2
+		LUA_INCLUDE ?= $(INCROOT)/include/lua5.2
+		LUA_LIB ?= -llua5.2
 		TESTER = bin/wordgrinder-debug
 	endif
 
-	NCURSES_CFLAGS := $(shell pkg-config ncursesw --cflags)
-	NCURSES_LIB := $(shell pkg-config ncursesw --libs)
-	X11_CFLAGS := $(shell pkg-config freetype2 --cflags) -I/usr/include/X11
-	X11_LIB := -lX11 -lXft $(shell pkg-config freetype2 --libs) 
+	ifndef NCURSES_CFLAGS
```

Comment:
Oh, yeah! I need a := variable to force evaluation right then.

(I _loathe_ make.)

Thanks!